core/graph/src/test/java/uk/gov/gchq/gaffer/graph/GraphTest.java

```diff
@@ -293,25 +288,13 @@ public void shouldConstructGraphFromSchemaURI() throws IOException, URISyntaxExc
         final Schema expectedSchema = new Schema.Builder()
                 .json(StreamUtil.elementsSchema(getClass()), StreamUtil.typesSchema(getClass()))
                 .build();
-        Graph graph = null;
-        File schemaDir = null;
-
-        try {
-            schemaDir = createSchemaDirectory();
-
-            // When
-            graph = new Graph.Builder()
-                    .config(new GraphConfig.Builder()
-                            .graphId(GRAPH_ID)
-                            .build())
-                    .storeProperties(storeInputUri)
-                    .addSchemas(typeInputUri, schemaInputUri)
-                    .build();
-        } finally {
-            if (schemaDir != null) {
-                FileUtils.deleteDirectory(schemaDir);
```

Comment:
I removed this because of that, for some reason it kept trying to delete the directory even though it wasn't there. So it was throwing an error because there was nothing to delete.